### DISCUSSION TITLE
Lake of Kalandra tablet suggestion

### DISCUSSION THREAD

**Discussion by guiltyhollow** [Lailloken/Exile-UI#93] [Ideas & Feedback]
Hello, I saw your post on Reddit about the tablet QoL and it was really impressive.
I would like to ask if it is possible to add a feature to highlight the tile selection on the right side of the tablet by the tile image.
for example, make the tile our build can't play such as reflect show up in red, and the good tile such as desired league mechanic or swap tile show up as green or something around this idea.

![Capture](https://user-images.githubusercontent.com/10905825/185553533-8b804bf6-ca03-4999-b399-ac3817010d1a.jpg)

thank you very much for your contribution to the community.

**Comment by Lailloken** [comment]
> thank you very much for your contribution to the community

You're welcome, and thanks for your suggestion.

I don't expect there to be tiles that are hard-counters to certain builds, but we'll have to see. It's also only a choice between three options, so I'll have to weigh how much effort it takes to implement and how much it improves gameplay. I have not improved my image-recognition system yet, so the user would also have to screen-cap the icons first to use this suggested feature. And I don't know if it's worth screen-capping, let's say, 20 different icons to make a choice between three options a bit quicker.

But I'll look into it once I have had some time with the new league.
